src/renderer/components/settings-electron.tsx

```diff
@@ -390,6 +391,25 @@ export class ElectronSettings extends React.Component<
           />
         </Tooltip>
       );
+    } else if (
+      process.platform === 'darwin' &&
+      process.arch === 'arm64' &&
+      semver.lt(version, '11.0.0')
```

Comment:
It might be better to put it in `utils/disable-download.ts`

```typescript
export function disableDownload(version: string) {
  return process.platform === 'darwin' && process.arch === 'arm64' && semver.lt(version, '11.0.0')
}
```

In the meantime, the relevant unit tests can be added:

```typescript
expect(disableDownload('10.0.0')).toBe(process.platform === 'darwin' && process.arch === 'arm64');
```

Reply:
Committed suggested recommendations